docs/DEV.md

```diff
@@ -1,5 +1,89 @@
 # Local dev with Go binary
 
+## Run the go binary with Azure and Rancher Desktop
```

Comment:
I would put this at the end of the file as it's not the main usecase

Reply:
done :)